Kaos-Frontend-Coding-Standards.md

```diff
@@ -74,7 +74,7 @@ At Sonatype we value the stability and maintainability of the code base while st
   * baz.view.html <-> baz.controller.js
 
 ## AngularJS Development
-* Each Angular Component should exist in it's own file
+* Each Angular Component should exist in it's own file, if it is independent
```

Comment:
I'm thinking something along the lines of - "All the honies making money, throw your hands up at me." Joke aside I think this should be clarified to anything that in injectable as it is what we discussed and causes no ambiguity.